pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -235,15 +247,20 @@ func (p *staticPolicy) validateState(s state.State) error {
 	// the set of CPUs stored in the state.
 	totalKnownCPUs := tmpDefaultCPUset.Clone()
 	tmpCPUSets := []cpuset.CPUSet{}
+	tmpAllCPUs := p.topology.CPUDetails.CPUs()
```

Comment:
nit: unnecessary duplication of `allCPUs` above.

Reply:
thanks, that's a good catch!